gallery_dl/extractor/hentainexus.py

```diff
@@ -42,15 +42,42 @@ def metadata(self, page):
             "title"      : extr('<h1 class="title">', '</h1>'),
             "artist"     : rmve(extr('viewcolumn">Artist</td>'     , '</td>')),
             "book"       : rmve(extr('viewcolumn">Book</td>'       , '</td>')),
+            "circle"     : rmve(extr('viewcolumn">Circle</td>'     , '</td>')),
+            "event"      : rmve(extr('viewcolumn">Event</td>'      , '</td>')),
             "language"   : rmve(extr('viewcolumn">Language</td>'   , '</td>')),
             "magazine"   : rmve(extr('viewcolumn">Magazine</td>'   , '</td>')),
             "parody"     : rmve(extr('viewcolumn">Parody</td>'     , '</td>')),
             "publisher"  : rmve(extr('viewcolumn">Publisher</td>'  , '</td>')),
             "description": rmve(extr('viewcolumn">Description</td>', '</td>')),
         }
         data["lang"] = util.language_to_code(data["language"])
+        if 'doujin' in data["tags"]:
+            data["is_doujinshi"] = True
+        else:
+            data["is_doujinshi"] = False
```

Comment:
This can be simplified to `data["is_doujinshi"] = 'doujin' in data["tags"]`
or, even better, rename this field to `type` and set it to Doujinshi or Manga like it is already done for other hmanga sites like pururin.io etc: `data["type"] = "Doujinshi" if 'doujin' in data["tags"] else "Manga"`

Reply:
thx for you suggestions.
btw why using double quotes in dict? is this just a personal practice or sth. more important?